features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml

```diff
@@ -1881,6 +1881,26 @@
         <Scopes>internal_workflow_instance_delete</Scopes>
     </Resource>
 
+    <!-- VC Credential Configuration Management API -->
```

Comment:
Don't we need to comment these too?

Reply:
No. access controls are come from pipeline